core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/StackV4RequestToStackConverter.java

```diff
@@ -349,39 +350,79 @@ private void validateStackAuthentication(StackV4Request source) {
         return convertedSet;
     }
 
-    private Set<LoadBalancer> createLoadBalancers(StackV4Request source, Stack stack) {
+    private Set<LoadBalancer> createLoadBalancers(StackV4Request source, Stack stack, DetailedEnvironmentResponse environment) {
```

Comment:
I think it would be useful to add some extra log lines, as there are a several if and it would help us investigate later

Reply:
Please review this method again and let me know if the new log statements cover everything you think we'll need.